src/components/Quote.js

```diff
@@ -0,0 +1,61 @@
+import { Component } from "react"
+import styled from "styled-components"
+
+const _QuoteContainer = styled.div.attrs({
+  className: ({ displayClasses }) =>
+    `w-100 sans flex-column fixed justify-center items-center ph4 pv4 bg-white shadow-2 z-2 tc ${displayClasses}`,
+})`
+  min-height: var(--height-4);
+  transition: 0.5s;
+`
+
+const _Quote = styled.div.attrs({
+  className: "font-4 bold dark-gray mb2",
+})``
+
+const _Author = styled.div.attrs({
+  className: "dark-gray o-50",
+})``
+
+class Quote extends Component {
+  state = {
+    displayClasses: "flex up-0rem",
+  }
+
+  componentDidMount() {
+    const { welcomeFlow } = this.props
+
+    if (welcomeFlow) return this.setState({ displayClasses: "dn" })
+    setTimeout(this.hideQuoteShowBot, 3000)
+  }
+
+  hideQuoteShowBot = () => {
+    const { initBot } = this.props
+    initBot()
+    this.setState({
+      displayClasses: "flex o-0 pointer-events-none up-8rem",
+    })
+  }
+
+  render() {
+    const {
+      props: {
+        quote: { quote, author },
+      },
+      state: { displayClasses },
+      hideQuoteShowBot,
+    } = this
+
```

Comment:
At the moment an error is thrown when there is no quote, think something like `if (!quote) return </div>` should do the trick?